src/routes/decks/[marketplaceDeckId]/+page.svelte

```diff
@@ -0,0 +1,98 @@
+<script lang="ts">
+  import { user, willAttemptLogin } from "$lib/firebase";
+  import { enhance } from "$app/forms";
+  import type { PageData } from "./$types";
+  import { afterUpdate } from "svelte";
+  import ProBadgeWrapper from "$lib/components/ProBadgeWrapper.svelte";
+  import { goto } from "$app/navigation";
+  export let data: PageData;
+
+  $: {
+    if (!$user && !willAttemptLogin())
+      goto(`/login?successRedirect=${encodeURIComponent(location.pathname)}`);
+  }
+
+  let isPro = false;
+  let isLoading = true;
+
+  afterUpdate(async () => {
+    const idToken = await $user?.getIdToken();
+    const userData = await fetch("/api/user", {
+      headers: { Authorization: `Bearer ${idToken}` },
+    });
+    isPro = (await userData.json()).isSubscribedToBlendPro;
+    isLoading = false;
+  });
+</script>
+
+<svelte:head>
+  <title>Deck Import</title>
+</svelte:head>
+
+<main>
+  <div class="content">
+    {#if isLoading}
```

Comment:
Added a simple loading label because the stripe API do be takin a long time

Reply:
We'll need a user data context that fetches this in the background after login and keeps it in memory, that can be part of the enhanced pro wrapper

Reply:
I'll make a story for this one